tests/test_losses_power_curve.py

```diff
@@ -228,6 +229,39 @@ def get_item_patch(self, key):
     assert any(og_power_curve != new_power_curve)
 
 
+@pytest.mark.parametrize('config', SAM_FILES[0:2])
+def test_power_curve_losses_mixin_class_wind_resource_too_high(config):
+    """Test mixin class behavior when wind resource is too high. """
+
+    with open(config, 'r') as fh:
+        sam_config = json.load(fh)
+
+    og_power_curve = np.array(sam_config["wind_turbine_powercurve_powerout"])
+
+    # patch required for 'wind_resource_data' access below
+    def get_item_patch(self, key):
+        return self.sam_sys_inputs.get(key)
+    PowerCurveLossesMixin.__getitem__ = get_item_patch
+
+    mixin = PowerCurveLossesMixin()
+    mixin.sam_sys_inputs = copy.deepcopy(sam_config)
+    mixin.sam_sys_inputs[PowerCurveLossesMixin.POWER_CURVE_CONFIG_KEY] = {
+        'target_losses_percent': 10,
+        'transformation': 'horizontal_translation'
+    }
+    mixin.sam_sys_inputs['wind_resource_data'] = {
+        'data': [(1, 10, 1_000, 0) for __ in BASIC_WIND_RES]
```

Comment:
Would be good to do a similar test for low wind speeds and/or a mix of the two, but if the PR is urgent, I won't impose

Reply:
Yep agreed, will add!